en/micro-integrator/docs/references/mediators/script-Mediator.md

```diff
@@ -9,6 +9,8 @@ The **Script Mediator** is used to invoke the functions of a variety of scripti
     -   If the original message consists of repetitive sections, you can use the [Iterate mediator](iterate-Mediator.md) to generate a relatively
     small payload using those repetitive sections. This will then allow you to use the Script mediator.
     -   The Script Mediator supports using Nashorn to execute JavaScripts, in addition to its default Rhino engine.
+    -   In order for Nashorn Engine to have fully native JSON compatibility with Script Mediator, you may need at least **OpenJDK or Oracle JDK version 8u112 or above**. 
```

Comment:
Shouldn't this go under 'Prerequisites'? 
Because this **Note** section seems to be related to 'large JSON data volumes'.

Reply:
Done.